extensions/adobe/experience/adcloud/experienceevent-all.schema.json

```diff
@@ -40,6 +40,21 @@
           "title": "Stitch ID Chosen",
           "type": "string",
           "description": "ID from the Ad Servers through AdCloud STATS to track click-through conversion on browsers that block third party cookies"
+        },
```

Comment:
We need to add the following attribute to the object
"meta:intendedToExtend": ["https://ns.adobe.com/xdm/context/experienceevent"]  This tells the system that this object should be treated as a mixin that is intended for schemas based on the experience event class